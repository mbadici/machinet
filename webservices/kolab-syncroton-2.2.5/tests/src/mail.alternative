MIME-Version: 1.0
content-class:
From: 
Subject: eee
Date: Sun, 2 Sep 2012 11:41:14 +0200
Importance: normal
X-Priority: 3
To: User <user@domain.tld>
Content-Type: multipart/alternative;
	boundary="_3EE5AD33-49F4-2808-9917-6969FE639CA7_"

--_3EE5AD33-49F4-2808-9917-6969FE639CA7_
Content-Transfer-Encoding: quoted-printable
Content-Type: text/plain; charset="iso-8859-1"

tt=

--_3EE5AD33-49F4-2808-9917-6969FE639CA7_
Content-Transfer-Encoding: quoted-printable
Content-Type: text/html; charset="iso-8859-1"

<HTML><HEAD><META HTTP-EQUIV=3D'Content-Type' CONTENT=3D'text/html; charset=
=3Diso-8859-1'></HEAD><BODY><SPAN style=3D'FONT-SIZE: 10pt; FONT-FAMILY: Ar=
ial; FONT-WEIGHT:Normal;'>tt</SPAN></BODY></HTML>=

--_3EE5AD33-49F4-2808-9917-6969FE639CA7_--

