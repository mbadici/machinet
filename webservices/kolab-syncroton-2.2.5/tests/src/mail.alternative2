Date: Thu, 16 Aug 2012 07:38:43 +0000
Subject: Re: test
Message-ID: <1h4e6qv8o4ib0mhoj2hmyj0s.1345102723450@email.android.com>
From: user@domain.tld
To: Kolab User <kolab@domain.tld>
MIME-Version: 1.0
Content-Type: multipart/alternative; boundary="--_com.android.email_15071870071490"

----_com.android.email_15071870071490
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: base64

Q3NkZnNkZnNkZgoKIkEuTC5FLkMiIDxhbGVjQGFsZWMucGw+IHdyb3RlOgoKPnRlc3QgLS0gQWxl
a3NhbmRlciAnQS5MLkUuQycgTWFjaG5pYWsgTEFOIE1hbmFnZW1lbnQgU3lzdGVtIERldmVsb3Bl
ciBbaHR0cDovL2xtcy5vcmcucGxdIFJvdW5kY3ViZSBXZWJtYWlsIERldmVsb3BlciBbaHR0cDov
L3JvdW5kY3ViZS5uZXRdIC0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0t
LS0tLS0tLS0tLSBQR1A6IDE5MzU5REMxIEBAIEdHOiAyMjc1MjUyIEBAIFdXVzogaHR0cDovL2Fs
ZWMucGwg
----_com.android.email_15071870071490
Content-Type: text/html; charset=utf-8
Content-Transfer-Encoding: base64

Q3NkZnNkZnNkZjxicj48YnI+JnF1b3Q7QS5MLkUuQyZxdW90OyAmbHQ7YWxlY0BhbGVjLnBsJmd0
OyB3cm90ZTo8YnI+PGJyPjxwcmU+dGVzdAoKLS0gCkFsZWtzYW5kZXIgJ0EuTC5FLkMnIE1hY2hu
aWFrCkxBTiBNYW5hZ2VtZW50IFN5c3RlbSBEZXZlbG9wZXIgW2h0dHA6Ly9sbXMub3JnLnBsXQpS
b3VuZGN1YmUgV2VibWFpbCBEZXZlbG9wZXIgIFtodHRwOi8vcm91bmRjdWJlLm5ldF0KLS0tLS0t
LS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tClBHUDogMTkzNTlE
QzEgQEAgR0c6IDIyNzUyNTIgQEAgV1dXOiBodHRwOi8vPHNwYW4gc3R5bGU9ImJhY2tncm91bmQt
Y29sb3I6ICNmZmZmMDAiPmFsZWM8L3NwYW4+LnBsCjwvcHJlPg==
----_com.android.email_15071870071490--
