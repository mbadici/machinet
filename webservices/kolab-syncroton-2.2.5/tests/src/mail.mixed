Date: Thu, 09 Aug 2012 13:18:31 +0000
Subject: Fwd: test html xx
Message-ID: <qma5x35ckynysjn2ee1jwwn8.1344518311869@email.android.com>
From: user@domain.tld
To: kolab@domain.tld
MIME-Version: 1.0
Content-Type: multipart/mixed; boundary="BOUNDARY"

--BOUNDARY
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: base64

ZWVl
--BOUNDARY
Content-Transfer-Encoding: 8bit
Content-Type: text/plain

aaa
--BOUNDARY
Content-Transfer-Encoding: base64
Content-Type: text/plain

YWFh
--BOUNDARY--
