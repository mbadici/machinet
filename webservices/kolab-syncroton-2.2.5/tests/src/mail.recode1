X-Sieve: CMU Sieve 2.4
X-Virus-Scanned: amavisd-new at mx.domain.de
Message-ID: <52CDD362.2020009@domain.de>
Date: Wed, 08 Jan 2014 23:38:26 +0100
From: Daniel <daniel@domain.de>
MIME-Version: 1.0
To: <alec@domain.de>
Subject: Bitte um =?ISO-8859-15?Q?R=FCckruf?=
Content-Type: text/plain; charset=ISO-8859-15; format=flowed
Content-Transfer-Encoding: 8bit

Hallo Björn,
bitte rufe mich doch mal ganz dringend zurück. 0178-5217029

Liebe Grüße
Daniel
