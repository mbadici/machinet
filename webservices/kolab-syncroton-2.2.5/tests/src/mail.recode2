Message-ID: <52B02B8F.9010500@domain.de>
Date: Tue, 17 Dec 2013 11:46:39 +0100
From: Sender <sender@domain.de>
User-Agent: Mozilla/5.0 (Windows; U; Windows NT 5.1; de; rv:1.9.2.13) Gecko/20101207 Thunderbird/3.1.7
To: Verborgene_Empfaenger:;
Subject: Weihnachtsbrief
Content-Type: multipart/mixed;
 boundary="------------000801030509090203070207"

This is a multi-part message in MIME format.
--------------000801030509090203070207
Content-Type: multipart/alternative;
 boundary="------------060305070900000101080707"


--------------060305070900000101080707
Content-Type: text/plain; charset=ISO-8859-15; format=flowed
Content-Transfer-Encoding: 8bit

Herzliche Adventsgrüße

--------------060305070900000101080707
Content-Type: text/html; charset=ISO-8859-15
Content-Transfer-Encoding: 8bit

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  <head>
    <meta http-equiv="content-type" content="text/html; charset=ISO-8859-15">
  </head>
  <body bgcolor="#ffffff" text="#000000">
    Herzliche Adventsgrüße<br>
  </body>
</html>

--------------060305070900000101080707--

--------------000801030509090203070207
Content-Type: application/pdf;
 name="Weihnachtsbrief13.pdf"
Content-Transfer-Encoding: base64
Content-Disposition: attachment;
 filename="Weihnachtsbrief13.pdf"

JVBERi0 [...shortened...] DYKJSVFT0YK
--------------000801030509090203070207--