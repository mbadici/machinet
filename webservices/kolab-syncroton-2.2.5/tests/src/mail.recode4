MIME-Version: 1.0
Content-Type: multipart/alternative;
 boundary="=_421c567043d5df058671910ddb659fc5"
Date: Thu, 06 Feb 2014 13:59:33 +0000
From: alec <test@test.cc>
To: <test2@test.cc>
Subject: test html + image
Message-ID: <6c821c802bc5293e179f9bcb5d4d8279@klab.cc>
User-Agent: Roundcube Webmail/1.0-git

--=_421c567043d5df058671910ddb659fc5
Content-Transfer-Encoding: 8bit
Content-Type: text/plain; charset=UTF-8

 

test message with inline image żąść 

 
--=_421c567043d5df058671910ddb659fc5
Content-Type: multipart/related;
 boundary="=_61b68141aaee8dc5b321e6059b43bb78"

--=_61b68141aaee8dc5b321e6059b43bb78
Content-Transfer-Encoding: quoted-printable
Content-Type: text/html; charset=UTF-8

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN">
<html><body style=3D'font-size: 10pt; font-family: Verdana,Geneva,sans-seri=
f'>
<p>test message with inline image =C5=BC=C4=85=C5=9B=C4=87</p>
<p><img src=3D"cid:139169517352f395455c016176663627@test.cc" alt=3D"logo.gif" /></p>
</body></html>

--=_61b68141aaee8dc5b321e6059b43bb78
Content-Transfer-Encoding: base64
Content-ID: <139169517352f395455c016176663627@test.cc>
Content-Type: image/jpeg;
 name=logo.gif
Content-Disposition: inline;
 filename=logo.gif;
 size=25745

/9j/4AAQSkZJRgABAgEASABIAAD/4QqARXhpZgAATU0AKgAAAAgABwESAAMAAAABAAEAAAEaAAUA
--=_61b68141aaee8dc5b321e6059b43bb78--

--=_421c567043d5df058671910ddb659fc5--
