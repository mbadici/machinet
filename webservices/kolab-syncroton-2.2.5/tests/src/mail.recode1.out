X-Sieve: CMU Sieve 2.4
X-Virus-Scanned: amavisd-new at mx.domain.de
Message-ID: <52CDD362.2020009@domain.de>
Date: Wed, 08 Jan 2014 23:38:26 +0100
From: Daniel <daniel@domain.de>
MIME-Version: 1.0
To: <alec@domain.de>
Subject: Bitte um =?ISO-8859-15?Q?R=FCckruf?=
Content-Type: text/plain; charset=ISO-8859-15; format=flowed
Content-Transfer-Encoding: base64

SGFsbG8gQmr2cm4sDQpiaXR0ZSBydWZlIG1pY2ggZG9jaCBtYWwgZ2FueiBkcmluZ2VuZCB6dXL8
Y2suIDAxNzgtNTIxNzAyOQ0KDQpMaWViZSBHcvzfZQ0KRGFuaWVs
